<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-K5UJHKQ" failures="1" tests="8" name="com.BHL.TC001_LoginPageTest" time="14.137" errors="0" timestamp="2025-03-21T15:25:26 IST" skipped="0">
  <testcase classname="com.BHL.TC001_LoginPageTest" name="assertionAll" time="0.006">
    <failure message="The following asserts failed:
	Welcome to
BHL GROUP’S expected [Welcome t
BHL GROUP’S] but found [Welcome to
BHL GROUP’S]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Welcome to
BHL GROUP’S expected [Welcome t
BHL GROUP’S] but found [Welcome to
BHL GROUP’S]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.BHL.TC001_LoginPageTest.assertionAll(TC001_LoginPageTest.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassConfigurations(TestMethodWorker.java:238)
at org.testng.internal.invokers.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:223)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:137)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- assertionAll -->
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC_001_verify_WelcomeText_On_LoginPage" time="0.099"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC_002_verify_Login_Button_Clickable" time="1.385"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC003_verify_ErrorMessageForEmailField" time="0.403"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC004_verify_ErrorMessageForPasswordField" time="0.132"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC_005_verify_Login" time="5.102"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC_006_LogoutFlow" time="3.397"/>
  <testcase classname="com.BHL.TC001_LoginPageTest" name="TC_007_reloginToBHL" time="3.613"/>
</testsuite> <!-- com.BHL.TC001_LoginPageTest -->
